dashboard/app/controllers/pd/workshop_material_orders_controller.rb

```diff
@@ -0,0 +1,50 @@
+require 'state_abbr'
```

Comment:
It's used in the view to populate the dropdown. I'll move the require to that file so it's more clear.